lib/filterx/object-dict-interface.c

```diff
@@ -191,6 +191,9 @@ _format_and_append_dict_elem(FilterXObject *key, FilterXObject *value, gpointer
   GString *json = (GString *) args[0];
   gboolean *first = (gboolean *) args[1];
 
+  if(!value)
```

Comment:
I think we shouldn't allow null-pointers in dict values (FilterX null values are okay).